src/custom/state/claim/hooks/index.ts

```diff
@@ -145,7 +146,7 @@ export function useUserClaims(account: Account): UserClaims | null {
       return
     }
 
-    fetchClaims(account)
+    fetchClaims(account, chainId)
```

Comment:
nit: you know how i feel about these types of function args, would prefer an object as the others are made but i can live with this